src/sources/LVLYieldSource.sol

```diff
@@ -0,0 +1,157 @@
+// SPDX-License-Identifier: BSL
+pragma solidity ^0.8.13;
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import { IYieldSource } from "../interfaces/IYieldSource.sol";
+import { ILvlStaking, UserInfo } from "../interfaces/lvl/ILvlStaking.sol";
+import { ILvlRouter } from "../interfaces/lvl/ILvlRouter.sol";
+import { ILvlPool } from "../interfaces/lvl/ILvlPool.sol";
+
+library LVLConstants {
+    uint256 public constant NETWORK_BNB = 0;
+    uint256 public constant NETWORK_ARBITRUM = 1;
+    
+    uint256 public constant TOKEN_LVL = 0;
```

Comment:
nit can we make these different from the values above, so there's no chance of mis-use/collision